config/ember-try.js

```diff
@@ -36,9 +36,12 @@ module.exports = async function () {
         npm: {
           devDependencies: {
             'ember-source': await getChannelURL('canary'),
+            '@glint/template': '*',
```

Comment:
not needed as listed in `package.json#devDependencies`?

Reply:
removed